frontend/views/components/ContributionItem.vue

```diff
@@ -0,0 +1,152 @@
+<template lang='pug'>
+.c-contribution-item
+  i(:class='iconClass')
+
+  template(v-if='type==="MONETARY" && hasWhoElse')
+    transition(name='replacelist')
+      div(
+        v-if='isVisible'
+        key='visible'
+      )
+        .c-contribution-list(v-html='listOfName')
+
+        i18n.is-unstyled.is-link-inherit.link(
+          tag='button'
+          type='button'
+          @click='isVisible = !isVisible'
+        ) Hide
+
+      div(
+        v-else
+        key='hidden'
+        @click='isVisible = !isVisible'
+        v-html='contribution'
+      )
+
+  span.has-text-bold(
+    v-else-if='type==="NON_MONETARY" && action === "GIVING"'
+  ) {{what}}
```

Comment:
This component (lines 26-28) can and should be removed, and similarly on line 5 you don't need the `v-if`. Use a computed property to generate the string and pass in via `v-html` (as you are already doing).